### PR TITLE
Features

### PR DESCRIPTION
**Pull request by punixcorn**
testing 
more comments 
changed formmatter

### CODE REVIEW

File: .installPacker.sh
Comment (Moonixp) [inline .installPacker.sh:1]: make this an installer script ?

File: lua/plugs.lua
Comment (Moonixp) [inline lua/plugs.lua:1]: Telescope and formatting still not working

**Review by Moonixp — CHANGES_REQUESTED** [review]
make installer script & fix telescope and formatting and we can merge

File: .installPacker.sh
Comment (punixcorn) [reply .installPacker.sh:1]: sure

File: lua/keymap.lua [hunk 1]
```diff
@@ -27,18 +30,19 @@ vim.keymap.set("v", "p", '"_dP')
...
+vim.keymap.set("n", "<A-k>", ":move .-2<CR>==")
+vim.keymap.set("v", "<A-j>", ":move '>+1<CR>gv=gv")
+vim.keymap.set("v", "<A-k>", ":move '<-2<CR>gv=gv")
 
 -- Disable arrow keys
-vim.keymap.set("n", "<Left>", ":echo 'Use h'<CR>")
-vim.keymap.set("n", "<Right>", ":echo 'Use l'<CR>")
-vim.keymap.set("n", "<Up>", ":echo 'Use k'<CR>")
-vim.keymap.set("n", "<Down>", ":echo 'Use j'<CR>")
+vim.keymap.set("n", "<Left>", ":echo 'Use h ya twat'<CR>")
```
Comment (Moonixp) [inline lua/keymap.lua:39]: really?

Comment (punixcorn) [reply lua/keymap.lua:39]: yes

File: plugin/nvimtree.lua [hunk 2]
```diff
@@ -0,0 +1,6 @@
+-- Nvim Tree
+vim.g.loaded_netrw = 1 -- required
+vim.g.loaded_netrwPlugin = 1 -- required
+vim.opt.termguicolors = true -- required
```
Comment (Moonixp) [inline plugin/nvimtree.lua:4]: some of these are already in settings.lua and could have put the rest there

File: plugin/null-ls.lua
Comment (Moonixp) [inline plugin/null-ls.lua:1]: didn't say anything about null-ls

File: plugin/nvimtree.lua (hunk 2 above)
Comment (punixcorn) [reply plugin/nvimtree.lua:4]: ok...

File: plugin/null-ls.lua
Comment (punixcorn) [reply plugin/null-ls.lua:1]: forgot, not needed, but does not interfere

**Review by Moonixp — APPROVED** [review]
ok adding it

**Pull request merged**